lib/rabbit_feed/publishing_options.rb

```diff
@@ -0,0 +1,42 @@
+module RabbitFeed
+  class PublishingOptions
```

Comment:
I am not sure I understand this object very well. I think it holds the headers for a while... so maybe this is nonsense but Why it needs to live on its own as it seems to be 1 to 1 related to the event and make no sense without it... Can't it be created when creating the event?

`Event.new name, payload, event_definition` and then extract this object (if needed) internally?